index.js

```diff
@@ -2,3 +2,5 @@ exports.client = require('./client');
 exports.cluster = require('./cluster');
 exports.process = require('./process');
 exports.server = require('./server');
+exports.ws_channel = require('./ws-channel');
```

Comment:
Not camel case. `ws_channel` is a constructor, right? Maybe `Websocket` instead of `ws?

Sorry I didn't do it before you started, but I added the eslint/jscs config files we are spreading across nodeops so we don't have to play "guess the code style".